Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
+      guard (~> 2.1)
+      guard-compat (~> 1.1)
+      rspec (>= 2.99.0, < 4.0)
+    listen (3.7.1)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    lumberjack (1.2.8)
+    method_source (1.0.0)
+    mock_redis (0.31.0)
+      ruby2_keywords
+    nenv (0.3.0)
+    notiffany (0.1.3)
+      nenv (~> 0.1)
+      shellany (~> 0.0)
+    pry (0.14.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    rake (13.0.6)
+    rb-fsevent (0.11.1)
```

Comment:
### [rb-fsevent](http://rubygems.org/gems/rb-fsevent)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.9.2 → 0.11.1

[_~~change-log~~, [source-code](https://github.com/thibaudgg/rb-fsevent), [gem-diff](https://my.diffend.io/gems/rb-fsevent/0.9.2/0.11.1)_]





<details>
<summary>Commits</summary>

A change of **93** commits. See the full changes on [the compare page](https://github.com/guard/rb-fsevent/compare/v0.9.2...v0.11.1).

These are the first 10 commits:
- (a80be21) [Update README.rdoc](https://github.com/guard/rb-fsevent/commit/a80be21b94ba2c9dbaa742e35b383c9907062653)
- (6334971) [Most of these changes were made many months ago and never co…](https://github.com/guard/rb-fsevent/commit/6334971ec735bb03a03ebf140f208239ffe9178f)
- (d9481b5) [change default build deployment target to 10.6](https://github.com/guard/rb-fsevent/commit/d9481b535d5bbe634a65d7ab6ce5bd74b9eb61d9)
- (75f1b9c) [Handle potential race condition when stopping](https://github.com/guard/rb-fsevent/commit/75f1b9cd9d75fe8d76725d7a268f1a52f401ba69)
- (fa8ad08) [Merge pull request #40 from rubinius/master](https://github.com/guard/rb-fsevent/commit/fa8ad081838d03cd5a54e29ec5be7481ba41ca44)
- (1ddd3df) [remove xcode project, replace with rakefile. xcodebuild stil…](https://github.com/guard/rb-fsevent/commit/1ddd3dfe61a1eb84c7cb9ebb0de90c64b4d92ced)
- (6d66f2e) [1.8 makes me grumpy](https://github.com/guard/rb-fsevent/commit/6d66f2e10072910a408ddba8b52f74d0617a631d)
- (a942a09) [rebuild fsevent_watch pre-compiled binary on a 32bit imac wh…](https://github.com/guard/rb-fsevent/commit/a942a09ea16d4acc8dbef7b862babfede85f67d6)
- (4cf18f4) [update README](https://github.com/guard/rb-fsevent/commit/4cf18f4e701712c79a7b1aa6c18339ff1ccc1e78)
- (f62d127) [various tweaks](https://github.com/guard/rb-fsevent/commit/f62d127a194a23dca011ee67ad4790dec2448c16)


</details>